app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  before_save :update_likes
```

Comment:
- [ ] You can use the after_save to call the #update_like_counter method every time a post has been liked